lib/utils/isStandardSyntaxAtRule.js

```diff
@@ -7,17 +7,21 @@
  * @param {atRule} postcss at-rule node
  * @return {boolean} If `true`, the declaration is standard
  */
-module.exports = function(atRule /*: postcss$atRule*/) /*: boolean*/ {
+module.exports = function(atRule /*: Object*/) /*: boolean*/ {
```

Comment:
Perhaps someone better than me at Flow can help out with this.

Reply:
I have no clue about Flow, to be honest. Maybe new properties should be added to this lines?

https://github.com/stylelint/stylelint/blob/6f86f23fe568daa6185d282079358ffc1a26fb54/flow-typed/postcss.js#L30-L38

Reply:
@CAYdenberg Any chance you've got a spare moment to check on this Flow issue please?